core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -266,7 +266,7 @@ class PlaintextConsumerTest extends BaseConsumerTest {
   def testSeek() {
     val consumer = this.consumers(0)
     val totalRecords = 50L
-    sendRecords(totalRecords.toInt)
+    sendCompressedRecords(totalRecords.toInt, tp)
```

Comment:
We should probably also test with uncompressed messages - if you make the above change to `sendRecords` you can either make the compression-codec a parameterized argument of the test or just send both compressed and uncompressed in a single run.